src/components/implementation/tests/pci/doc.md

```diff
@@ -0,0 +1,9 @@
+## tests
+
+This is the skeleton interface used by the `mkcomponent.sh` script to aid in the creation of a new component.
```

Comment:
Add some documentation here. Doesn't have to be much. But you can at least add a Warning about the hard-coded checking for the devices. Can repeat the qemu version etc... here.

Particularly relevant for "assumptions" below.